SampleProject/VOKCoreDataManagerTests/ManagedObjectImportExportTests.m

```diff
@@ -597,6 +599,34 @@ - (void)testUniqueKeyPath
     XCTAssertEqualObjects([[self customDateFormatter] stringFromDate:person.birthDay], dateString);
 }
 
+- (void)testImportingTwoObjectsWithTheSameUniqueIDOverwritesWithUniqueKey
```

Comment:
is it worth adding a test to check that any import after this will trigger an assert? [line 310](https://github.com/designatednerd/Vokoder/blob/major_version_changes/Pod/Classes/VOKCoreDataManager.m#L310)